### PR TITLE
Fix conda cuda incompatibility and upgrade to torch 2.0 and cuda 11.8

### PR DESCRIPTION
**Pull request by alexsax**
## Motivation and Context

Installing the environment.yml file somehow causes pytorch, pytorch3d, pytorch-cuda, and cuda-version to use different versions:
In the image below, it shows the cuda version that each package is using:
- Pytorch: 11.2
- Pytorch-cuda: 11.7
- cudaversion: 11.8
- Pytorch3d: 11.7

image

This breaks loading PyTorch3D, but not Pytorch:
I can verify that in the conda environment, torch is using 11.2:
```
>>> torch.version.cuda
'11.2'
```
And it works for torch! But breaks when using pytorch3d:
```
>>> from pytorch3d.vis.plotly_vis import plot_scene, AxisArgs
Traceback (most recent call last):
 .../pytorch3d/renderer/blending.py", line 10, in <module>
    from pytorch3d import _C
ImportError: libtorch_cuda_cu.so: cannot open shared object file: No such file or directory
```

The environment.yml file suggests reason for that old pytorch version is [pytorch3d compatibility](https://github.com/facebookresearch/home-robot/blob/b643698a372c9b64af8995d7295d28ba8b7dfe99/src/home_robot/environment.yml#L11). But P3D supports pytorch 2.0.1 now, so this patch upgrades home-robot to using pytorch 2.0.1. This resolves the incompatibility issue
## Changelog

Update environment.yml files in home-robot and home-robot-hw. 
Update README.md to reflect new cuda version

## How Has This Been Tested

Fresh conda install and loaded pytorch3d + pytorch
## Types of changes

- [x] Docs change / refactoring / dependency upgrade
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

# Checklist:

- [x] I have made changes to existing documentation where needed.
- [ ] I have added tests that show that the PR is functional.
- [ ] I ran `tests/hw_manual_test.py` on hardware and observed that robot behavior looks normal.

### CODE REVIEW

**Comment by cpaxton** [comment]
@yvsriram do you see any reason why this would break our evaluations?

**Comment by yvsriram** [comment]
> @yvsriram do you see any reason why this would break our evaluations?

I don't think they should, but I will try to create a new sim environment and confirm today.

**Comment by cpaxton** [comment]
It looks like there's a problem with habitat sim install.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by cpaxton** [comment]
@alexsax 
```
Warning: you have pip-installed dependencies in your environment file, but you do not list pip itself as one of your conda dependencies.  Conda may not use the correct pip to install your packages, and they may end up in the wrong place.  Please add an explicit pip dependency.  I'm adding one for you, but still nagging you.
conda-forge/linux-64                                        Using cache
conda-forge/noarch                                          Using cache


Looking for: ['pip', 'withbullet', 'habitat-sim-ovmm-challenge-2023']


warning  libmamba Added empty dependency for problem type SOLVER_RULE_UPDATE
Could not solve for environment specs
The following packages are incompatible
├─ [32meigenpy[0m is installable with the potential options
│  ├─ [32meigenpy [1.5.6|1.5.7|...|2.2.0][0m would require
│  │  └─ [32mpython >=2.7,<2.8.0a0 [0m, which can be installed;
│  ├─ [32meigenpy [1.5.6|1.5.7|...|2.6.8][0m would require
│  │  └─ [32mpython >=3.6,<3.7.0a0 [0m, which can be installed;
│  ├─ [32meigenpy [1.5.6|1.5.7|...|2.7.9][0m would require
│  │  └─ [32mpython >=3.7,<3.8.0a0 [0m, which can be installed;
│  ├─ [32meigenpy [1.6.11|1.6.12|...|3.1.0][0m would require
│  │  └─ [32mpython >=3.8,<3.9.0a0 [0m, which can be installed;
│  ├─ [32meigenpy [2.5.0|2.6.0][0m would require
│  │  └─ [32mboost >=1.72.0,<1.72.1.0a0 [0m with the potential options
│  │     ├─ [32mboost [1.72.0|1.74.0][0m would require
│  │     │  └─ [32mpython >=3.6,<3.7.0a0 [0m, which can be installed;
│  │     ├─ [32mboost [1.72.0|1.74.0|1.78.0][0m would require
│  │     │  └─ [32mpython >=3.7,<3.8.0a0 [0m, which can be installed;
│  │     ├─ [32mboost [1.72.0|1.74.0|1.78.0][0m would require
│  │     │  └─ [32mpython >=3.8,<3.9.0a0 [0m, which can be installed;
│  │     ├─ [32mboost 1.72.0[0m would require
│  │     │  └─ [32mpython >=2.7,<2.8.0a0 [0m, which can be installed;
│  │     └─ [32mboost 1.72.0[0m would require
│  │        └─ [32mboost-cpp 1.72.0.* [0m with the potential options
│  │           ├─ [32mboost-cpp [1.72.0|1.74.0][0m would require
│  │           │  └─ [32micu >=68.1,<69.0a0 [0m, which can be installed;
│  │           ├─ [32mboost-cpp [1.72.0|1.74.0][0m would require
│  │           │  └─ [32micu >=69.1,<70.0a0 [0m, which can be installed;
│  │           ├─ [32mboost-cpp [1.72.0|1.74.0|1.78.0][0m would require
│  │           │  └─ [32micu >=70.1,<71.0a0 [0m, which can be installed;
│  │           ├─ [32mboost-cpp [1.72.0|1.74.0][0m would require
│  │           │  └─ [32micu >=67.1,<68.0a0 [0m, which can be installed;
│  │           └─ [32mboost-cpp 1.72.0[0m would require
│  │              └─ [32micu >=64.2,<65.0a0 [0m, which can be installed;
│  ├─ [32meigenpy [2.5.0|2.6.0|...|2.7.9][0m would require
│  │  └─ [32mboost >=1.74.0,<1.74.1.0a0 [0m with the potential options
│  │     ├─ [32mboost [1.74.0|1.78.0][0m would require
│  │     │  └─ [32mpython >=3.10,<3.11.0a0 [0m, which can be installed;
│  │     ├─ [32mboost [1.72.0|1.74.0][0m, which can be installed (as previously explained);
│  │     ├─ [32mboost [1.72.0|1.74.0|1.78.0][0m, which can be installed (as previously explained);
│  │     ├─ [32mboost [1.72.0|1.74.0|1.78.0][0m, which can be installed (as previously explained);
│  │     └─ [32mboost 1.74.0[0m would require
│  │        └─ [32mboost-cpp 1.74.0.* [0m, which can be installed (as previously explained);
│  ├─ [32meigenpy [2.6.10|2.6.11|...|3.1.0][0m would require
│  │  └─ [32mpython >=3.10,<3.11.0a0 [0m, which can be installed;
│  ├─ [32meigenpy [2.7.13|2.7.14|...|3.1.0][0m would require
│  │  └─ [32mboost >=1.78.0,<1.78.1.0a0 [0m with the potential options
│  │     ├─ [32mboost [1.74.0|1.78.0][0m, which can be installed (as previously explained);
│  │     ├─ [32mboost [1.72.0|1.74.0|1.78.0][0m, which can be installed (as previously explained);
│  │     ├─ [32mboost [1.72.0|1.74.0|1.78.0][0m, which can be installed (as previously explained);
│  │     ├─ [32mboost 1.78.0[0m would require
│  │     │  └─ [32mpython >=3.11,<3.12.0a0 [0m, which can be installed;
│  │     └─ [32mboost 1.78.0[0m would require
│  │        └─ [32mboost-cpp 1.78.0.* [0m with the potential options
│  │           ├─ [32mboost-cpp [1.72.0|1.74.0|1.78.0][0m, which can be installed (as previously explained);
│  │           └─ [31mboost-cpp 1.78.0[0m would require
│  │              └─ [31micu >=72.1,<73.0a0 [0m, which conflicts with any installable versions previously reported;
│  └─ [32meigenpy [2.7.14|2.8.0|...|3.1.0][0m would require
│     └─ [32mpython >=3.11,<3.12.0a0 [0m, which can be installed;
├─ [32mhabitat-sim-ovmm-challenge-2023[0m is installable with the potential options
│  ├─ [32mhabitat-sim-ovmm-challenge-2023 0.2.4[0m would require
│  │  └─ [32mwithbullet <0 [0m, which can be installed;
│  └─ [32mhabitat-sim-ovmm-challenge-2023 0.2.4[0m would require
│     ├─ [32mlibxcb >=1.15,<1.16.0a0 [0m, which can be installed;
│     └─ [32mxorg-libxinerama >=1.1.5,<1.2.0a0 [0m, which requires
│        └─ [32mlibgcc-ng >=12 [0m, which requires
│           └─ [32m_libgcc_mutex 0.1 conda_forge[0m, which can be installed;
├─ [32mharfbuzz[0m is installable with the potential options
│  ├─ [32mharfbuzz [1.0.6|1.3.4][0m would require
│  │  └─ [32micu 56.* [0m, which can be installed;
│  ├─ [32mharfbuzz [1.3.4|1.4.3|1.7.1|1.7.6][0m would require
│  │  └─ [32micu 58.* [0m with the potential options
│  │     ├─ [31micu 58.2[0m conflicts with any installable versions previously reported;
│  │     └─ [32micu 58.1[0m, which can be installed;
│  ├─ [31mharfbuzz [1.5.0|1.7.4|...|4.3.0][0m would require
│  │  └─ [31micu >=58.2,<59.0a0 [0m, which conflicts with any installable versions previously reported;
│  ├─ [32mharfbuzz 2.4.0[0m would require
│  │  └─ [32micu >=64.2,<65.0a0 [0m, which can be installed;
│  ├─ [32mharfbuzz [2.4.0|2.7.0|2.7.1|2.7.2][0m would require
│  │  └─ [32micu >=67.1,<68.0a0 [0m, which can be installed;
│  ├─ [32mharfbuzz [2.7.2|2.7.3|...|3.0.0][0m would require
│  │  └─ [32micu >=68.1,<69.0a0 [0m, which can be installed;
│  ├─ [32mharfbuzz [3.1.0|3.1.1][0m would require
│  │  └─ [32micu >=68.2,<69.0a0 [0m, which can be installed;
│  ├─ [32mharfbuzz [3.1.1|3.1.2|...|4.2.0][0m would require
│  │  └─ [32micu >=69.1,<70.0a0 [0m, which can be installed;
│  ├─ [32mharfbuzz [4.2.0|4.2.1|4.3.0|4.4.0|4.4.1][0m would require
│  │  └─ [32micu >=70.1,<71.0a0 [0m, which can be installed;
│  └─ [31mharfbuzz [5.0.1|5.1.0|...|7.3.0][0m conflicts with any installable versions previously reported;
├─ [32mjpeg[0m is requested and can be installed;
├─ [32mopencv[0m is installable with the potential options
│  ├─ [32mopencv [2.4.12|2.4.13|...|4.2.0][0m would require
│  │  ├─ [32mpy-opencv [3.4.7 py27h5ca1d4c_5|3.4.7 py27h5ca1d4c_7|...|4.2.0 py27h5ca1d4c_2][0m, which requires
│  │  │  └─ [32mpython >=2.7,<2.8.0a0 [0m, which can be installed;
│  │  └─ [32mpython [2.7* |>=2.7,<2.8.0a0 ][0m, which can be installed;
│  ├─ [32mopencv 3.1.0[0m would require
│  │  └─ [32mpython 3.4* [0m, which can be installed;
│  ├─ [32mopencv [3.1.0|3.2.0|...|3.4.3][0m would require
│  │  └─ [32mpython [3.5* |>=3.5,<3.6.0a0 ][0m, which can be installed;
│  ├─ [32mopencv [3.1.0|3.2.0|...|4.5.3][0m would require
│  │  ├─ [32mpython 3.6* [0m, which can be installed;
│  │  └─ [32mpython_abi 3.6.* *_cp36m[0m, which requires
│  │     └─ [32mpython 3.6.* [0m, which can be installed;
│  ├─ [32mopencv [3.3.1|3.4.1|...|4.5.2][0m would require
│  │  ├─ [32mpy-opencv [3.4.7 py36h5ca1d4c_7|3.4.7 py36h5ca1d4c_8|...|4.5.2 py36he2ddec3_0][0m, which requires
│  │  │  └─ [32mpython >=3.6,<3.7.0a0 [0m, which can be installed;
│  │  └─ [32mpython >=3.6,<3.7.0a0 [0m, which can be installed;
│  ├─ [32mopencv [3.4.1|3.4.2|...|4.6.0][0m would require
│  │  ├─ [32mpy-opencv [3.4.7 py37h5ca1d4c_4|3.4.7 py37h5ca1d4c_6|...|4.6.0 py37hf05f0b3_5][0m, which requires
│  │  │  └─ [32mpython >=3.7,<3.8.0a0 [0m, which can be installed;
│  │  └─ [32mpython >=3.7,<3.8.0a0 [0m, which can be installed;
│  ├─ [32mopencv [3.4.7|3.4.8|...|4.8.0][0m would require
│  │  ├─ [32mpy-opencv [3.4.7 py38h5ca1d4c_7|3.4.7 py38h5ca1d4c_8|...|4.8.0 py38h5e64f72_0][0m, which requires
│  │  │  └─ [32mpython >=3.8,<3.9.0a0 [0m, which can be installed;
│  │  └─ [32mpython >=3.8,<3.9.0a0 [0m, which can be installed;
│  ├─ [31mopencv 4.2.0[0m would require
│  │  └─ [31mlibopencv 4.2.0 py36_5[0m, which does not exist (perhaps a missing channel);
│  ├─ [32mopencv [4.4.0|4.5.0|...|4.5.4][0m would require
│  │  ├─ [32mharfbuzz >=2.4.0,<3.0a0 [0m with the potential options
│  │  │  ├─ [31mharfbuzz [1.5.0|1.7.4|...|4.3.0][0m, which cannot be installed (as previously explained);
│  │  │  ├─ [32mharfbuzz 2.4.0[0m, which can be installed (as previously explained);
│  │  │  ├─ [32mharfbuzz [2.4.0|2.7.0|2.7.1|2.7.2][0m, which can be installed (as previously explained);
│  │  │  └─ [32mharfbuzz [2.7.2|2.7.3|...|3.0.0][0m, which can be installed (as previously explained);
│  │  └─ [32mlibopencv [4.4.0 py39_3|4.5.0 py39_0|...|4.5.3 py39h70bf20d_1][0m, which requires
│  │     └─ [32mharfbuzz >=2.4.0,<3.0a0 [0m with the potential options
│  │        ├─ [31mharfbuzz [1.5.0|1.7.4|...|4.3.0][0m, which cannot be installed (as previously explained);
│  │        ├─ [32mharfbuzz 2.4.0[0m, which can be installed (as previously explained);
│  │        ├─ [32mharfbuzz [2.4.0|2.7.0|2.7.1|2.7.2][0m, which can be installed (as previously explained);
│  │        └─ [32mharfbuzz [2.7.2|2.7.3|...|3.0.0][0m, which can be installed (as previously explained);
│  ├─ [31mopencv 4.5.0[0m would require
│  │  └─ [31mlibopencv 4.5.0 py36_5[0m, which does not exist (perhaps a missing channel);
│  ├─ [31mopencv 4.5.0[0m would require
│  │  └─ [31mlibopencv 4.5.0 py36_6[0m, which does not exist (perhaps a missing channel);
│  ├─ [32mopencv [4.5.3|4.5.4|...|4.8.0][0m would require
│  │  ├─ [32mpy-opencv [4.5.3 py310hfdc917e_8|4.5.3 py31hfdc917e_6|...|4.8.0 py310hfdc917e_0][0m, which requires
│  │  │  └─ [32mpython >=3.10,<3.11.0a0 [0m, which can be installed;
│  │  └─ [32mpython >=3.10,<3.11.0a0 [0m, which can be installed;
│  ├─ [32mopencv 4.5.3[0m would require
│  │  └─ [32mlibopencv [4.5.3 py39ha7f30a5_5|4.5.3 py39hd5eeab7_4][0m, which requires
│  │     └─ [32mharfbuzz >=3.0.0,<4.0a0 [0m with the potential options
│  │        ├─ [31mharfbuzz [1.5.0|1.7.4|...|4.3.0][0m, which cannot be installed (as previously explained);
│  │        ├─ [32mharfbuzz [2.7.2|2.7.3|...|3.0.0][0m, which can be installed (as previously explained);
│  │        ├─ [32mharfbuzz [3.1.0|3.1.1][0m, which can be installed (as previously explained);
│  │        └─ [32mharfbuzz [3.1.1|3.1.2|...|4.2.0][0m, which can be installed (as previously explained);
│  ├─ [32mopencv [4.5.3|4.5.5][0m would require
│  │  └─ [32mlibopencv [4.5.3 py39h7d09d5f_7|4.5.3 py39ha7f30a5_6|4.5.5 py39h7d09d5f_0][0m, which requires
│  │     └─ [32mharfbuzz >=3.1.1,<4.0a0 [0m with the potential options
│  │        ├─ [31mharfbuzz [1.5.0|1.7.4|...|4.3.0][0m, which cannot be installed (as previously explained);
│  │        ├─ [32mharfbuzz [3.1.0|3.1.1][0m, which can be installed (as previously explained);
│  │        └─ [32mharfbuzz [3.1.1|3.1.2|...|4.2.0][0m, which can be installed (as previously explained);
│  ├─ [32mopencv 4.5.5[0m would require
│  │  └─ [32mlibopencv [4.5.5 py39h12858e4_8|4.5.5 py39h12858e4_9|...|4.5.5 py39hfb30bf4_6][0m, which requires
│  │     └─ [32mharfbuzz [>=4.0.1,<5.0a0 |>=4.1.0,<5.0a0 |>=4.2.0,<5.0a0 ][0m with the potential options
│  │        ├─ [31mharfbuzz [1.5.0|1.7.4|...|4.3.0][0m, which cannot be installed (as previously explained);
│  │        ├─ [32mharfbuzz [3.1.1|3.1.2|...|4.2.0][0m, which can be installed (as previously explained);
│  │        └─ [32mharfbuzz [4.2.0|4.2.1|4.3.0|4.4.0|4.4.1][0m, which can be installed (as previously explained);
│  ├─ [32mopencv 4.5.5[0m would require
│  │  └─ [32mlibopencv [4.5.5 py39ha3d0060_11|4.5.5 py39hd011c1b_11][0m, which requires
│  │     └─ [32mqt-main >=5.15.3,<5.16.0a0 [0m with the potential options
│  │        ├─ [32mqt-main 5.15.3[0m would require
│  │        │  ├─ [32malsa-lib >=1.2.3,<1.2.4.0a0 [0m, which can be installed;
│  │        │  ├─ [32mgst-plugins-base >=1.20.1,<1.21.0a0 [0m, which can be installed;
│  │        │  └─ [32mpulseaudio >=14.0,<14.1.0a0 [0m with the potential options
│  │        │     ├─ [32mpulseaudio 14.0[0m, which can be installed;
│  │        │     └─ [31mpulseaudio 14.0[0m would require
│  │        │        └─ [31malsa-lib >=1.2.8,<1.2.9.0a0 [0m, which conflicts with any installable versions previously reported;
│  │        ├─ [32mqt-main [5.15.3|5.15.4][0m would require
│  │        │  ├─ [32malsa-lib >=1.2.3.2,<1.2.4.0a0 [0m, which can be installed;
│  │        │  ├─ [32mgst-plugins-base >=1.20.2,<1.21.0a0 [0m, which can be installed;
│  │        │  └─ [32mpulseaudio >=14.0,<14.1.0a0 [0m with the potential options
│  │        │     ├─ [32mpulseaudio 14.0[0m, which can be installed;
│  │        │     └─ [31mpulseaudio 14.0[0m, which cannot be installed (as previously explained);
│  │        ├─ [32mqt-main 5.15.4[0m would require
│  │        │  ├─ [32malsa-lib >=1.2.6.1,<1.2.7.0a0 [0m, which can be installed;
│  │        │  ├─ [32mgst-plugins-base >=1.20.2,<1.21.0a0 [0m, which can be installed;
│  │        │  └─ [32mpulseaudio >=14.0,<14.1.0a0 [0m with the potential options
│  │        │     ├─ [32mpulseaudio 14.0[0m, which can be installed;
│  │        │     └─ [31mpulseaudio 14.0[0m, which cannot be installed (as previously explained);
│  │        ├─ [32mqt-main 5.15.6[0m would require
│  │        │  ├─ [32malsa-lib >=1.2.7.2,<1.2.8.0a0 [0m, which can be installed;
│  │        │  ├─ [32mgst-plugins-base >=1.20.3,<1.21.0a0 [0m, which can be installed;
│  │        │  └─ [32mpulseaudio >=14.0,<14.1.0a0 [0m with the potential options
│  │        │     ├─ [32mpulseaudio 14.0[0m, which can be installed;
│  │        │     └─ [31mpulseaudio 14.0[0m, which cannot be installed (as previously explained);
│  │        ├─ [31mqt-main 5.15.6[0m would require
│  │        │  ├─ [31malsa-lib >=1.2.8,<1.2.9.0a0 [0m, which conflicts with any installable versions previously reported;
│  │        │  ├─ [32mgst-plugins-base >=1.21.1,<1.22.0a0 [0m with the potential options
│  │        │  │  ├─ [31mgst-plugins-base [1.21.2|1.21.3|1.22.0][0m would require
│  │        │  │  │  └─ [31mlibxcb >=1.13,<1.14.0a0 [0m, which conflicts with any installable versions previously reported;
│  │        │  │  ├─ [32mgst-plugins-base 1.21.1[0m, which can be installed;
│  │        │  │  └─ [32mgst-plugins-base 1.21.1[0m would require
│  │        │  │     └─ [32malsa-lib >=1.2.7.2,<1.2.8.0a0 [0m, which can be installed;
│  │        │  └─ [31mlibxcb >=1.13,<1.14.0a0 [0m, which conflicts with any installable versions previously reported;
│  │        ├─ [31mqt-main 5.15.6[0m would require
│  │        │  ├─ [31malsa-lib >=1.2.8,<1.2.9.0a0 [0m, which conflicts with any installable versions previously reported;
│  │        │  ├─ [31mgst-plugins-base [>=1.21.2,<1.22.0a0 |>=1.21.3,<1.22.0a0 ][0m, which cannot be installed (as previously explained);
│  │        │  └─ [31mlibxcb >=1.13,<1.14.0a0 [0m, which conflicts with any installable versions previously reported;
│  │        ├─ [31mqt-main 5.15.8[0m would require
│  │        │  └─ [31mlibjpeg-turbo >=2.1.5.1,<3.0a0 [0m, which requires
│  │        │     └─ [31mjpeg <0.0.0a [0m, which conflicts with any installable versions previously reported;
│  │        └─ [31mqt-main 5.15.8[0m would require
│  │           ├─ [31malsa-lib >=1.2.8,<1.2.9.0a0 [0m, which conflicts with any installable versions previously reported;
│  │           ├─ [31mgst-plugins-base >=1.22.0,<1.23.0a0 [0m but there are no viable options
│  │           │  ├─ [31mgst-plugins-base 1.22.3[0m conflicts with any installable versions previously reported;
│  │           │  ├─ [31mgst-plugins-base 1.22.0[0m would require
│  │           │  │  ├─ [31malsa-lib >=1.2.8,<1.2.9.0a0 [0m, which conflicts with any installable versions previously reported;
│  │           │  │  └─ [31mlibxcb >=1.13,<1.14.0a0 [0m, which conflicts with any installable versions previously reported;
│  │           │  ├─ [31mgst-plugins-base 1.22.3[0m would require
│  │           │  │  ├─ [31malsa-lib >=1.2.8,<1.2.9.0a0 [0m, which conflicts with any installable versions previously reported;
│  │           │  │  └─ [32mlibxcb >=1.15,<1.16.0a0 [0m, which can be installed;
│  │           │  ├─ [31mgst-plugins-base 1.22.3[0m would require
│  │           │  │  └─ [31malsa-lib >=1.2.8,<1.2.9.0a0 [0m, which conflicts with any installable versions previously reported;
│  │           │  ├─ [31mgst-plugins-base 1.22.4[0m would require
│  │           │  │  ├─ [31malsa-lib >=1.2.9,<1.2.10.0a0 [0m, which conflicts with any installable versions previously reported;
│  │           │  │  └─ [32mgstreamer 1.22.4 h98fc4e7_0[0m, which can be installed;
│  │           │  ├─ [31mgst-plugins-base 1.22.4[0m would require
│  │           │  │  ├─ [31malsa-lib >=1.2.9,<1.2.10.0a0 [0m, which conflicts with any installable versions previously reported;
│  │           │  │  └─ [32mgstreamer 1.22.4 h98fc4e7_1[0m, which can be installed;
│  │           │  ├─ [31mgst-plugins-base 1.22.5[0m would require
│  │           │  │  ├─ [31malsa-lib >=1.2.9,<1.2.10.0a0 [0m, which conflicts with any installable versions previously reported;
│  │           │  │  └─ [32mgstreamer 1.22.5 h98fc4e7_0[0m, which can be installed;
│  │           │  └─ [31mgst-plugins-base [1.21.2|1.21.3|1.22.0][0m, which cannot be installed (as previously explained);
│  │           └─ [31mlibxcb >=1.13,<1.14.0a0 [0m, which conflicts with any installable versions previously reported;
│  ├─ [32mopencv [4.5.5|4.6.0][0m would require
│  │  └─ [32mlibopencv [4.5.5 py39ha3d0060_12|4.5.5 py39ha3d0060_13|...|4.6.0 py39hf4bb9d8_2][0m, which requires
│  │     └─ [32mqt-main >=5.15.4,<5.16.0a0 [0m with the potential options
│  │        ├─ [32mqt-main [5.15.3|5.15.4][0m, which can be installed (as previously explained);
│  │        ├─ [32mqt-main 5.15.4[0m, which can be installed (as previously explained);
│  │        ├─ [32mqt-main 5.15.6[0m, which can be installed (as previously explained);
│  │        ├─ [31mqt-main 5.15.6[0m, which cannot be installed (as previously explained);
│  │        ├─ [31mqt-main 5.15.6[0m, which cannot be installed (as previously explained);
│  │        ├─ [31mqt-main 5.15.8[0m, which cannot be installed (as previously explained);
│  │        └─ [31mqt-main 5.15.8[0m, which cannot be installed (as previously explained);
│  ├─ [32mopencv 4.5.5[0m would require
│  │  └─ [32mlibopencv [4.5.5 py39h7d09d5f_1|4.5.5 py39h7d09d5f_2|4.5.5 py39hd0eba51_3|4.5.5 py39hd0eba51_4][0m, which requires
│  │     └─ [32mharfbuzz [>=3.3.1,<4.0a0 |>=3.4.0,<4.0a0 ][0m, which can be installed (as previously explained);
│  ├─ [32mopencv [4.6.0|4.7.0|4.8.0][0m would require
│  │  ├─ [32mpy-opencv [4.6.0 py311h781c19f_6|4.6.0 py311h781c19f_7|...|4.8.0 py311h781c19f_0][0m, which requires
│  │  │  └─ [32mpython >=3.11,<3.12.0a0 [0m, which can be installed;
│  │  └─ [32mpython >=3.11,<3.12.0a0 [0m, which can be installed;
│  ├─ [32mopencv [4.5.3|4.6.0|4.7.0][0m would require
│  │  └─ [32mlibopencv [4.5.3 py39h0b4d57a_8|4.6.0 py39h596d152_5|...|4.7.0 py39hb375605_0][0m, which requires
│  │     └─ [32mqt-main >=5.15.6,<5.16.0a0 [0m with the potential options
│  │        ├─ [32mqt-main 5.15.6[0m, which can be installed (as previously explained);
│  │        ├─ [31mqt-main 5.15.6[0m, which cannot be installed (as previously explained);
│  │        ├─ [31mqt-main 5.15.6[0m, which cannot be installed (as previously explained);
│  │        ├─ [31mqt-main 5.15.8[0m, which cannot be installed (as previously explained);
│  │        └─ [31mqt-main 5.15.8[0m, which cannot be installed (as previously explained);
│  ├─ [31mopencv [4.7.0|4.8.0][0m would require
│  │  └─ [31mlibopencv [4.7.0 py39h01ea3e4_4|4.7.0 py39h0293f97_2|...|4.8.0 py39hea38a9e_0][0m, which requires
│  │     └─ [31mlibjpeg-turbo >=2.1.5.1,<3.0a0 [0m, which cannot be installed (as previously explained);
│  └─ [31mopencv [4.5.5|4.6.0][0m would require
│     └─ [31m_openmp_mutex >=5.1 [0m, which requires
│        └─ [31m_libgcc_mutex 0.1 main[0m, which conflicts with any installable versions previously reported;
├─ [31mpulseaudio-daemon[0m is not installable because there are no viable options
│  ├─ [31mpulseaudio-daemon 16.1[0m would require
│  │  ├─ [31malsa-lib >=1.2.9,<1.2.10.0a0 [0m, which conflicts with any installable versions previously reported;
│  │  └─ [31mpulseaudio 16.1 *_4[0m, which conflicts with any installable versions previously reported;
│  └─ [31mpulseaudio-daemon 16.1[0m would require
│     └─ [31malsa-lib >=1.2.8,<1.2.9.0a0 [0m, which conflicts with any installable versions previously reported;
├─ [32mqt-main[0m is installable with the potential options
│  ├─ [32mqt-main 5.15.2[0m would require
│  │  ├─ [32malsa-lib >=1.2.3,<1.2.4.0a0 [0m, which can be installed;
│  │  ├─ [32mgst-plugins-base >=1.18.5,<1.19.0a0 [0m, which can be installed;
│  │  └─ [32mpulseaudio >=14.0,<14.1.0a0 [0m with the potential options
│  │     ├─ [32mpulseaudio 14.0[0m, which can be installed;
│  │     └─ [31mpulseaudio 14.0[0m, which cannot be installed (as previously explained);
│  ├─ [32mqt-main 5.15.3[0m, which can be installed (as previously explained);
│  ├─ [32mqt-main [5.15.3|5.15.4][0m, which can be installed (as previously explained);
│  ├─ [32mqt-main 5.15.4[0m, which can be installed (as previously explained);
│  ├─ [32mqt-main 5.15.6[0m, which can be installed (as previously explained);
│  ├─ [31mqt-main 5.15.6[0m, which cannot be installed (as previously explained);
│  ├─ [31mqt-main 5.15.6[0m, which cannot be installed (as previously explained);
│  ├─ [31mqt-main 5.15.8[0m, which cannot be installed (as previously explained);
│  ├─ [31mqt-main 5.15.8[0m, which cannot be installed (as previously explained);
│  ├─ [31mqt-main 5.15.2[0m would require
│  │  ├─ [31mgst-plugins-base >=1.14.0,<1.15.0a0 [0m, which conflicts with any installable versions previously reported;
│  │  ├─ [31mgstreamer >=1.14.0,<1.15.0a0 [0m, which conflicts with any installable versions previously reported;
│  │  └─ [31micu >=58.2,<59.0a0 [0m, which conflicts with any installable versions previously reported;
│  └─ [31mqt-main 5.15.2[0m would require
│     ├─ [31mgst-plugins-base >=1.14.1,<1.15.0a0 [0m, which conflicts with any installable versions previously reported;
│     ├─ [31mgstreamer >=1.14.1,<1.15.0a0 [0m, which conflicts with any installable versions previously reported;
│     └─ [31micu >=58.2,<59.0a0 [0m, which conflicts with any installable versions previously reported;
└─ [31mwithbullet[0m is not installable because it conflicts with any installable versions previously reported.

```

**Comment by yvsriram** [comment]
Thanks for trying @cpaxton. Is this with a fresh environment install?

File: README.md [hunk 1]
```diff
@@ -73,12 +73,12 @@ Optionally, setup a [catkin workspace](docs/catkin.md) to use improved ROS visua
 
 #### 2. Run Install Script
 
-Make sure you have the correct environment variables set: `CUDA_HOME` should point to your cuda install, matching the one used by your python environment. We recommend 11.7, and it's what will be automatically installed above. You can download it from [nvidia's downloads page](https://developer.nvidia.com/cuda-11-7-0-download-archive?target_os=Linux&target_arch=x86_64&Distribution=Ubuntu). Download the runfile, and make sure to check the box NOT to install your drivers.
+Make sure you have the correct environment variables set: `CUDA_HOME` should point to your cuda install, matching the one used by your python environment. We recommend 11.8, and it's what will be automatically installed above. You can download it from [nvidia's downloads page](https://developer.nvidia.com/cuda-11-7-0-download-archive?target_os=Linux&target_arch=x86_64&Distribution=Ubuntu). Download the runfile, and make sure to check the box NOT to install your drivers.
```
Comment (cpaxton) [inline README.md:76]: We should update this link to point to 11.8

**Comment by cpaxton** [comment]
At least locally, this one works with torch = 2.0.0
maybe we should try this

**Comment by cpaxton** [comment]
@alexsax / @yvsriram I pushed an update here, looks like it works with 2.0.0; is this easier to install?

**Comment by yvsriram** [comment]
Yes will try it and let you know

**Comment by cpaxton** [comment]
This should be fixed; with fresh install:
```
(home-robot-only_200_118_v19) cpaxton@cpaxton-ThinkStation-P620:~/src/home-robot$ conda list | grep torch 
ffmpeg                    4.3                  hf484d3e_0    pytorch
pyg                       2.3.1           py39_torch_2.0.0_cu118    pyg
pytorch                   2.0.0           py3.9_cuda11.8_cudnn8.7.0_0    pytorch
pytorch-cluster           1.6.1           py39_torch_2.0.0_cu118    pyg
pytorch-cuda              11.8                 h7e8668a_5    pytorch
pytorch-mutex             1.0                        cuda    pytorch
pytorch-scatter           2.1.1           py39_torch_2.0.0_cu118    pyg
pytorch3d                 0.7.4           py39_cu118_pyt200    pytorch3d
torchaudio                2.0.0                py39_cu118    pytorch
torchtriton               2.0.0                      py39    pytorch
(home-robot-only_200_118_v19) cpaxton@cpaxton-ThinkStation-P620:~/src/home-robot$ 
```

**Comment by cpaxton** [comment]
robostack is messing it up in the hardware version, we might just want to drop that and install ROS via apt

**Pull request closed without merging**